### ISSUE TITLE
bit_write_CRC buffer overflow with bigPlan_2000.dwg

### ISSUE THREAD

**Issue by rurban** [LibreDWG/libredwg#315]
`./json  bigPlan_2000.dwg`
without any error

`./dwg  bigPlan_2000.json`
several bit_write_CRC buffer overflow errors

import into AutoCAD crashes

by timoria

**Comment by rurban** [comment]
```
ERROR: bit_write_CRC buffer overflow at pos 80846-80896, size 80896
ERROR: bit_write_CRC buffer overflow at pos 106446-106496, size 106496
ERROR: bit_write_CRC buffer overflow at pos 132038-132096, size 132096
ERROR: bit_write_CRC buffer overflow at pos 149454-149504, size 149504
ERROR: bit_write_CRC buffer overflow at pos 158670-158720, size 158720
ERROR: bit_write_CRC buffer overflow at pos 160718-160768, size 160768
ERROR: bit_write_CRC buffer overflow at pos 182222-182272, size 182272
ERROR: bit_write_CRC buffer overflow at pos 189390-189440, size 189440
ERROR: bit_write_CRC buffer overflow at pos 198606-198656, size 198656
ERROR: bit_write_CRC buffer overflow at pos 223182-223232, size 223232
ERROR: bit_write_CRC buffer overflow at pos 239566-239616, size 239616
ERROR: bit_write_CRC buffer overflow at pos 297933-297984, size 297984
ERROR: bit_write_CRC buffer overflow at pos 304077-304128, size 304128
ERROR: bit_write_CRC buffer overflow at pos 316365-316416, size 316416
ERROR: bit_write_CRC buffer overflow at pos 351182-351232, size 351232
ERROR: bit_write_CRC buffer overflow at pos 409550-409600, size 409600
ERROR: bit_write_CRC buffer overflow at pos 506830-506880, size 506880
ERROR: bit_write_CRC buffer overflow at pos 545741-545792, size 545792
ERROR: bit_write_CRC buffer overflow at pos 548814-548864, size 548864
```

with innocent LINE objects

**Comment by rurban** [comment]
Fixed the problem in the `smoke/write_CRC` branch. Simple stupidity on my side.
See here https://ci.appveyor.com/project/rurban/libredwg/builds/37886823

**Comment by rurban** [comment]
Fixed with 0.12.2
